File: WebExtension/src/VisitorGift/Models/VisitItem.ts
export interface VisitItem {
    code: string;

    name: string;

    recipient: string;
    
    recipientName: string;

    comment: string;

    count: string;

    source: ItemSource;
}

export enum ItemSource
{
    Defective = 0,
    Common = 1,
    Limited = 2,
    Special = 3
}

File: Visit/Models/VisitItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TrinityHelp.Visit.Models
{
    public class VisitItem
    {
        public string Code { get; set; }

        public string Name { get; set; }

        public Guid Recipient { get; set; }

        public string RecipientName { get; set; }

        public string Comment { get; set; }

        public int Count { get; set; }

        public ItemSource Source { get; set; }
    }
}
